web/src/components/Settings/WebhookSection.tsx

```diff
@@ -79,7 +79,7 @@ const WebhookSection = () => {
                 {webhooks.map((webhook) => (
                   <tr key={webhook.id}>
                     <td className="whitespace-nowrap px-3 py-2 text-sm text-gray-900 dark:text-gray-400">{webhook.name}</td>
-                    <td className="whitespace-nowrap px-3 py-2 text-sm text-gray-900 dark:text-gray-400">{webhook.url}</td>
+                    <td className="max-w-[200px] px-3 py-2 text-sm text-gray-900 dark:text-gray-400 truncate">{webhook.url}</td>
```

Comment:
```suggestion
                    <td className="max-w-[200px] px-3 py-2 text-sm text-gray-900 dark:text-gray-400 truncate" title={webhook.url}>{webhook.url}</td>
```

Reply:
thanks for suggesting the change, no worries. I will make it.